imagej/imagej-ops2-tutorial/docs/OpCurrency.md

```diff
@@ -0,0 +1,132 @@
+# Writing Small, Easy, Modular Ops
+
+SciJava Ops is designed for modularity, extensibility, and granularity - by writing your Ops on the smallest functional elements possible, you'll allow SciJava Ops to lift your Op to new image types, enable parallelism, and more without any additional effort!
+
+## Element-wise Ops
+
+Simple pixel-wise operations like addition, inversion, and more can be written on a single pixel (i.e. `RealType`) - therefore, ImageJ Ops2 takes care to automagically adapt pixel-wise Ops across a wide variety of image types. Therefore, if you would like to write a pixel-wise Op, we recommend the following structure.
```

Comment:
Use another word for the second "Therefore" or replace with "Thus".

Reply:
I chose to just remove it in 60d278fb